packages/client-app/src/components/DataTable/DataTableCell.vue

```diff
@@ -11,7 +11,7 @@ withDefaults(
 <template>
   <component
     :is="is"
-    class="min-h-[31px] min-w-0 border-b-[.5px] border-r-[.5px] border-r-[inherit] text-sm last:border-r-0 group-last:border-b-transparent"
+    class="min-h-[31px] h-[31px] min-w-0 border-b-[.5px] border-r-[.5px] border-r-[inherit] text-sm last:border-r-0 group-last:border-b-transparent"
```

Comment:
I strongly suggest we don't use pixel values!!! Its an anti-pattern

Reply:
just snap to the grid 😄

Reply:
@antlio added this 👀 can we add a lint rule for this?

Reply:
Well there are rare occasions where it's ok. But like 99% if the time we shouldn't. Just need to get all the devs on board!

If we are doing it a lot then our config is wrong and it should be added there.

Reply:
not disagreeing but snapping the grid wouldnt maintain height consistency here.

i'll give a global eye on this and take another path to reach both 👍

Reply:
Then we adjust some other spots. If everything is lined up then you don't need use any pixel values